container/lxd/lxd.go

```diff
@@ -143,6 +145,7 @@ func (manager *containerManager) CreateContainer(
 		return
 	}
 
+	// TODO This might be dead code. Do we always get len(nics) > 0?
```

Comment:
Please add a "juju" TODO - TODO(your nick) so that we'd know who to ask if questions arise.
Also this one looks like it might be worth a bug/a card/a tech-debt card... Please add one of these and reference it in TODO as well.